plugins/CoreHome/angularjs/common/services/piwik-api.js

```diff
@@ -105,32 +118,32 @@
                 headers: headers
             };
 
-            $http(ajaxCall).success(onSuccess).error(onError);
+            var promise = $http(ajaxCall).then(onSuccess, onError);
```

Comment:
We should not use `requestPromise` but `promise` as `requestPromise` gets never cancelled and we would go into success callback of any promise instead of the error callback when a request is aborted